sdk/src/feature_set.rs

```diff
@@ -277,6 +285,8 @@ lazy_static! {
         (full_inflation::bl::enable::id(), "Full inflation enabled by BL"),
         (full_inflation::bunghi::vote::id(), "Community vote allowing bunghi to enable full inflation"),
         (full_inflation::bunghi::enable::id(), "Full inflation enabled by bunghi"),
+        (full_inflation::buburuza::vote::id(), "Community vote allowing buburuza to enable full inflation"),
+        (full_inflation::buburuza::enable::id(), "Full inflation enabled by buburuza"),        
```

Comment:
The code prettier doesn't like this trailing whitespace
```suggestion
        (full_inflation::buburuza::enable::id(), "Full inflation enabled by buburuza"),
```